### PR TITLE
[SCAL-95706] add visibleActions param

### PR DESCRIPTION
**Pull request by cristian-cojocaru-ts**

### CODE REVIEW

File: src/embed/search.ts [hunk 1]
```diff
@@ -161,6 +162,9 @@ export class SearchEmbed extends TsEmbed {
         if (hiddenActions?.length) {
             queryParams[Param.HideActions] = hiddenActions;
         }
+        if (visibleActions?.length) {
+            queryParams[Param.VisibleActions] = visibleActions;
+        }
```
Comment (ashubham) [inline src/embed/search.ts:167]: I see action handling in all 3 flavors search/app/pinboard, can we converge these to be in the single place ?

Also, lets throw an error when the user uses both `visibleActions` and `hiddenActions`

**Review by ashubham — CHANGES_REQUESTED** [review]
PTAL at the comment

File: src/embed/search.ts (hunk 1 above)
Comment (kailash-choudhary-thoughtspot) [reply src/embed/search.ts:167]: since now we support Actions for each embed flavour, shall we add it in Base Class.

**Review by ashubham — APPROVED** [review]

File: src/embed/ts-embed.ts [hunk 2]
```diff
@@ -96,6 +96,11 @@ export interface ViewConfig {
      * (...), and the contextual menu.
      */
     hiddenActions?: Action[];
+    /**
+     * The list of actions to display from the primary menu, more menu
+     * (...), and the contextual menu.
+     */
```
Comment (ashubham) [inline src/embed/ts-embed.ts:102]: Can we put a tag like "Available since: 1.5.0"

**Pull request merged**